packages/create-pantheon-decoupled-kit/src/templates/partials/next-drupal/nextDrupalLayout.hbs

```diff
@@ -39,7 +39,9 @@ export default function Layout({ children, footerMenu, preview = false }) {
 			{{#if search}}
 			<div className={styles.searchHeaderContainer}>
 			<Header navItems={navItems} />
-				<SearchInput />
+				<div>
```

Comment:
Ideally we wouldn't need this inner div. I think a top margin on the `.container` in searchInput.module.css would achieve the same effect without extra markup.